### ISSUE TITLE
Improve field checks

### ISSUE THREAD

**Issue by schliesser** [ochorocho/syspass-addon#12]
https://github.com/ochorocho/syspass-addon/blob/cb4aedb22662ae8fee3d2afbb9975fc1b67e384b/javascript/syspass.js#L82
If the element is null this test will return true and an exception is thrown in the next line. Improvement:
```js
if(typeof(usernameField) != 'undefined' && usernameField != null){
```

Also before this line we need to check weither the passwordField could be found to prevent errors on pages without any password field:
 https://github.com/ochorocho/syspass-addon/blob/cb4aedb22662ae8fee3d2afbb9975fc1b67e384b/javascript/syspass.js#L79

**Comment by ochorocho** [comment]
Improved field checks 
https://github.com/ochorocho/syspass-addon/blob/master/javascript/syspass.js#L79

Already available in version 1.6.2

**Comment by ochorocho** [comment]
Fixed :-) Thanks
